tokens/boxShadow.json

```
@@ -0,0 +1,20 @@
+{
+  "box-shadow": {
+    "1": {
+      "value": "0 1px 4px rgb(0 0 0 / 22%), 0 1px 2px rgb(0 0 0 / 26%)",
```

Please also move the descriptions, like so:

```suggestion
      "$description": "Smallest elevation. Use for Cards + Buttons.",
      "value": "0 1px 4px rgb(0 0 0 / 22%), 0 1px 2px rgb(0 0 0 / 26%)",
```

---

Done in https://github.com/demos-europe/demosplan-ui/pull/193/commits/f3a4ac024df652350d44b5744a15e701a3378953

---

I meant to move all descriptions from the scss file, not only for this one variable.

---

@spiess-demos yes I have already moved it!